File: benchmark/results/ts-inline-sonnet/run-002/a-mcp/output/src/domain/analytics/services/metrics-aggregator.ts
import { AnalyticsEvent } from '../aggregates/event';

/** MetricsAggregator service. */
export class MetricsAggregator {

  create(input: Partial<AnalyticsEvent>): AnalyticsEvent { throw new Error('not implemented'); }

  findById(id: string): AnalyticsEvent | null { throw new Error('not implemented'); }

  list(limit: number): AnalyticsEvent[] { throw new Error('not implemented'); }

  delete(id: string): void { throw new Error('not implemented'); }
}

File: benchmark/results/java-invocation-inline/run-005/b-baseline/output/com/metaengine/demo/analytics/services/MetricsAggregator.java
package com.metaengine.demo.analytics.services;

import com.metaengine.demo.analytics.aggregates.Event;
import java.util.List;

/** Aggregates Event data into metrics. */
public class MetricsAggregator {

    /** Create a new aggregated Event from a partial input. */
    public Event create(Event input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find an aggregated Event by id; returns null if not found. */
    public Event findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List aggregated Events, capped at the supplied limit. */
    public List<Event> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete an aggregated Event by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}
